### ISSUE TITLE
It shouldn't affect code editors and IDEs (vscode, Sublime Text, IntelliJ and Xcode)

### ISSUE THREAD

**Issue by Ivanca** [rux616/karabiner-windows-mode#30]
I think there should be an alternative version of this plugin that make exceptions for code editors and IDEs, those apps have their own keybindings files (e.g. _keybindings.json_ in vscode) and changing ctrl for cmd for some combinations messes them up big time, for example it transforms `ctrl+f` to `cmd+f` but the command that the app had mapped for `ctrl+f` becomes completely inaccesible; plus another problem that arises is that the bindings stop making sense (because where it says `ctrl` now sometimes means `cmd`)

#32

**Comment by Ivanca** [comment]
☎️ 💢 ❓

**Comment by Ivanca** [comment]
😢 🙏

**Comment by rux616** [comment]
Closed by #37.

**Comment by rcdailey** [comment]
@rux616 can you help me understand how this resolves the issue? Maybe I'm too new to how this stuff gets set up, but right now "Control" isn't acting as Windows "Ctrl" in VS Code. I have to use the "Command" key for things like CTRL+SHIFT+P (on my MBP I have to press Command+Shift+P).

What am I missing?

**Comment by rcdailey** [comment]
I did read that part of the readme and I understood it to mean that somehow those shortcuts were dynamically disabled or excluded from those apps when they are in the foreground. So it seems like I just need to use application shortcuts to get what I want?

Thanks for the quick reply!

**Comment by rux616** [comment]
> I did read that part of the readme and I understood it to mean that somehow those shortcuts were dynamically disabled or excluded from those apps when they are in the foreground. So it seems like I just need to use application shortcuts to get what I want?
> 
> Thanks for the quick reply!

Sorry, I didn't mean to imply that you didn't read it, I was just pointing out where the info lives for anyone else who wishes to read it.

But yes, you are correct in that the standard shortcuts like CTRL+C are disabled in IDEs via lines like `k.condition('unless', bundle.standard, file_paths.standard)),` where `bundle.standard` includes a number of other bundles and is defined [here](https://github.com/rux616/karabiner-windows-mode/blob/edef4e9704da2b60a2ab54ff2dc0d4e664044d66/jsonnet/lib/bundle.libsonnet#L61) currently.

**Comment by rcdailey** [comment]
I see that firefox is in the list. But when I do Control+Shift+F to set that to an extension, the Command key actually gets thrown in there. Seems like those exclusions aren't working for me. I installed everything via `brew`, not sure if that's an issue. I still feel like I'm missing something. I enabled every single shortcut after importing them. Is that wrong?

And thank you again for the help. I apologize for sounding defensive in my last reply!
